Source/Scene/Model.js

```diff
@@ -4094,20 +4018,32 @@ define([
                 }
 
                 this._loadResources = new LoadResources();
-                parse(this);
             }
+            parse(this);
```

Comment:
Why did this move outside of the above `this._state !== ModelState.FAILED` check?